versions/3.1.0.md

```diff
@@ -2687,32 +2693,34 @@ Field Name | Type | Description
 
 This object MAY be extended with [Specification Extensions](#specificationExtensions).
 
-The discriminator object is legal only when using one of the composite keywords `oneOf`, `anyOf`, `allOf`.
+##### Discriminator Object Example
 
-In OAS 3.0, a response payload MAY be described to be exactly one of any number of types:
+A payload MAY be described to be exactly one of any number of types:
 
 ```yaml
-MyResponseType:
+MyType:
   oneOf:
+  - type: 'null'
```

Comment:
One special case is the "null" type, which is introduced in 3.1. I am suggesting to provide one "null" type example to present the implications of the "null" type when it is used along with a discriminator object and oneOf/anyOf. There is only one sentence in this 3.1 draft spec that mentions the "null" type. 

My understanding is this schema is supported, and it is the new recommended approach in lieu of the deprecated "nullable". But the "null" type does not have a discriminator field in its payload, so adding it as an example would help to clarify that it's ok to use.

On one hand, the "null" type does not have a discriminator value which breaks the normative statement about the discriminator being a required field. But on the other hand, it seems to be valid because of the following sentence: **When using the discriminator, inline schemas will not be considered.**